client/components/TestQueueRow/index.jsx

```diff
@@ -114,25 +115,44 @@ const TestQueueRow = ({
     };
 
     const renderAssignedUserToTestPlan = () => {
+        const gitShaDateDifference = (
```

Comment:
Matt felt strongly that the Test Queue rows should just show the date, not the Git SHA... Can we remove the Git SHA here, (but keep it where it appears in the NewTestPlanReportModal picker)?

Reply:
If it is just the date displayed then what would be a delineation if there are two tests published the same day?

Reply:
Yep, had the same thought--see sibling comment about adding time to date...

Reply:
+ 1 on adding time in addition to absolute date format. SHAs don't really mean anything whereas time solves the delineation issue brought up by Rich and does mean something for the user.